source

```diff
@@ -102551,7 +102551,8 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>attribute value (single-quoted) state</span>.</dd>
 
    <dt>U+003E GREATER-THAN SIGN (&gt;)</dt>
-   <dd><span>Parse error</span>. Treat it as per the "anything else" entry below.</dd>
+   <dd>This is a <dfn data-x="parse-error-missing-attribute-value">missing-attribute-value</dfn>
+   <span>parse error</span>. Switch to the data state. Emit the current tag token.</dd>
```

Comment:
Sorry, missed this one: `data state` should be enclosed in `<span>` like in all other occurrences of term.